components/ISCNForm.vue

```diff
@@ -9,11 +9,24 @@
       />
     </UFormGroup>
 
-    <UFormGroup label="Title" class="flex-1" required>
-      <UInput v-model="formData.title" placeholder="Enter ISCN title" />
+    <UFormGroup
+      label="Title"
+      :error="(!formData.title) && 'Title is required'"
+      class="flex-1 text-left"
+      required
+    >
+      <UInput
+        v-model="formData.title"
+        placeholder="Enter ISCN title"
+      />
     </UFormGroup>
 
-    <UFormGroup label="Description" class="flex-1" required>
+    <UFormGroup
+      label="Description"
+      class="flex-1 text-left"
+      :error="(!formData.description) && 'Description is required'"
```

Comment:
Unnecessary brackets
```suggestion
      :error="!formData.description && 'Description is required'"
```

Reply:
[🎨 Improve condition checks for required fields in ISCNForm](https://github.com/likecoin/nft-book-press/pull/352/commits/9f7b1286287b8c05cb457e002f367f1161c8eef3)